### PR TITLE
Fix for zeta issues.

### PR DESCRIPTION
**Pull request by aoifeboyle**
Response to issue #411

### CODE REVIEW

File: tardis/plasma/properties/atomic.py [hunk 1]
```diff
@@ -153,6 +153,8 @@ class ZetaData(BaseAtomicDataProperty):
     outputs = ('zeta_data',)
 
     def _filter_atomic_property(self, zeta_data, selected_atoms):
+        for temperature in range(42000, 82000, 2000):
```
Comment (wkerzendorf) [inline tardis/plasma/properties/atomic.py]: We should extrapolate and not increase the storage. I mean in the end how far it is outside -> outside means use the last valid value

**Comment by wkerzendorf** [comment]
@aoifeboyle instead of using `interpolate.interp1d` use this: `interpolate.UnivariateSpline(x, y, k=1, s=0, ext=3)` that should work to use the last values

**Comment by aoifeboyle** [comment]
@wkerzendorf Ok, I've had a look at that and it only works with 1-D arrays for the x and y values, so it will only work if I create a loop and interpolate individually for each ion. Is that alright?

**Comment by wkerzendorf** [comment]
@aoifeboyle - that's right. I'm just fixing this - I'll make a PR to your PR. I should be done momentarily. We can also skype if you want.

**Comment by wkerzendorf** [comment]
@aoifeboyle I made a PR to your PR. have a look and merge if happy.

**Pull request merged**